imports/ui/helpers.coffee

```diff
@@ -11,8 +11,8 @@ UI.registerHelper 'formatDateRange', (dateRange)->
   return formatDateRange(dateRange)
 
 UI.registerHelper 'incidentToText', (incident) ->
-  if @cases
-    incidentDescription = pluralize("case", @cases)
+  if incident.cases
```

Comment:
Why does the variable for the cases property need to change, but not the variable for the deaths and specify properties?